lambda_src/lambda_function.py

```diff
@@ -99,13 +102,23 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
     req_body = loads(event['body'])
 
     batch_id = headers[BATCH_ID_HEADER]
+    batch_id_exists = check_if_initialized(batch_id)
     write_uri = headers.get('write-uri')
     LOG.debug(f'sync_flow() received destination: {write_uri}.')
 
     if write_uri:
         destination_driver = import_module(
             f'geff.drivers.destination_{urlparse(write_uri).scheme}'
         )
+    else:
+        if not batch_id_exists:
+            initialize_dynamodb_item(batch_id) # For the first Lambda
+        else:
+            while get_response(batch_id) is None: # For the subsequent Lambda
+                sleep(5)
```

Comment:
what do we need `sleep` here?

Reply:
Had added it to keep checking for a response in the table, in case a response has not been returned by the time the second Lambda is triggered.

Reply:
Had added it to keep checking for a response in the table, in case a response has not been returned by the time the second Lambda is triggered.